encrypted-media/scripts/playback-temporary-multikey-sequential.js

```diff
@@ -37,13 +37,21 @@ function runTest(config,qualifier) {
         function onMessage(event) {
             config.messagehandler(event.messageType, event.message, {variantId: event.target.variantId}).then(function(response) {
                 return event.target.update(response);
+            }).then(function(){
+                if (!_waitingForKey) {
+                    _video.src = URL.createObjectURL(_mediaSource);
+                    _video.play();
+                } else {
+                    _waitingForKey = false;
```

Comment:
This is not guaranteed to be true. For example, multiple rounds might be necessary. (In practice, this probably only applies to the first message, which would fall into the if block instead.)